### PR TITLE
Enhancement for using AzCopy downloadable link from secret file

### PR DESCRIPTION
**Pull request by sireeshapotnuru**
Currently the downloadable link for AzCopy is hardcoded in the script, so enhanced to read the link from secret file for better maintainability.

### CODE REVIEW

**Comment by sireeshapotnuru** [comment]
Below are the test results 

LIS Version: lis-rpms-4.3.3
Distro: rhel_6.10
Running kernel version: 2.6.32-754.17.1.el6.x86_64

Loaded LIS Modules after reboot:
hv_vmbus: 4.3.3
hv_balloon: 4.3.3
hyperv_keyboard: 4.3.3
hv_netvsc: 4.3.3
hid_hyperv: 4.3.3
hv_utils: 4.3.3
hyperv_fb: 4.3.3
hv_storvsc: 4.3.3

File: scripts/rhel_validation/prepare_lis_vm.sh [hunk 1]
```diff
@@ -48,9 +49,13 @@ function prepare_vm {
 
     popd
 
-    wget https://aka.ms/downloadazcopy-v10-linux
-    tar xvf downloadazcopy-v10-linux
-    ./azcopy_linux_amd64_10.1.2/azcopy cp "${kernel_version}" "https://${storage_account}.blob.core.windows.net/kernel/${storage_sas_token}" --recursive
+    storage_account=$(echo $storage_account|sed -e 's/[\r\n]//g')
```
Comment (chvalean) [inline scripts/rhel_validation/prepare_lis_vm.sh]: why you need to sanitize the variables inside the script?

File: scripts/rhel_validation/prepare_lis_vm.sh [hunk 2]
```diff
@@ -48,9 +49,13 @@ function prepare_vm {
...
+    storage_sas_token=$(echo $storage_sas_token|sed -e 's/[\r\n]//g')
+    azcopy_downloadlink=$(echo $azcopy_downloadlink|sed -e 's/[\r\n]//g')
+
+    wget -O azcopy.tar.gz ${azcopy_downloadlink}
+    tar xvf azcopy.tar.gz
```
Comment (chvalean) [inline scripts/rhel_validation/prepare_lis_vm.sh]: remove verbose flag, not needed in automation

Comment (sireeshapotnuru) [reply scripts/rhel_validation/prepare_lis_vm.sh]: removed verbose flag

File: scripts/rhel_validation/prepare_lis_vm.sh (hunk 1 above)
Comment (sireeshapotnuru) [reply scripts/rhel_validation/prepare_lis_vm.sh]: when reading variables from secret file \r and  \n are appended to the values, so used sed to remove these characters

**Comment by sireeshapotnuru** [comment]
@chvalean please approve and merge this PR

**Pull request merged**